src/containers/Articles/Article.css

```diff
@@ -0,0 +1,23 @@
+.articleImage {
+    width: 100%;
+}
+.tag {
+    padding: 6px 3px;
+    border-radius: 5px; margin-right:5px;
+}
+.paragraph2{
+    margin:10px 0;
+}
+.capitalise{
+    text-transform: capitalize;
+}
+.articleMetaData{
+    margin:10px 0;
+    padding:10px 0;
+}
+.content_enter{
+    margin:0 auto;
+}
+.tag_grid{
+    display: inline-block
+}
```

Comment:
@ssewilliam, please add EOF here.

Reply:
@abulojoshua1,
- I have made changes here as well, please check this again

Reply:
ok